src/selectors/entities/users.ts

```diff
@@ -490,8 +490,9 @@ export function makeGetDisplayName(): (a: GlobalState, b: $ID<UserProfile>, c: b
         getTeammateNameDisplaySetting,
         (state, _, useFallbackUsername = true) => useFallbackUsername,
         getConfig,
-        (user, teammateNameDisplaySetting, useFallbackUsername, config) => {
-            const useAdminTemmateNameDisplaySetting = config.LockTeammateNameDisplay === 'true';
+        getLicense,
+        (user, teammateNameDisplaySetting, useFallbackUsername, config, license) => {
+            const useAdminTemmateNameDisplaySetting = license.LockTeammateNameDisplay === 'true' && config.LockTeammateNameDisplay === 'true';
```

Comment:
What if license or config is undefined? Actually the unit tests are failing exactly cause of that reason

Reply:
license and config are keys present on the initial global state so they should only even be undefined in mocks.